src/main/java/net/modfest/scatteredshards/api/impl/ShardLibraryPersistentState.java

```diff
@@ -36,9 +36,6 @@ public ShardLibraryPersistentState() {
 	
 	private static void addDefaultShardTypes() {
 		ShardLibrary library = ScatteredShardsAPI.getServerLibrary();
-		library.shardTypes().put(ScatteredShards.id("visitor"), ShardType.VISITOR);
```

Comment:
This is where you'd add server-data in, and if server-data isn't present, default to the visitor-challenge-secret trio.